clientv3/integration/kv_test.go

```diff
@@ -37,7 +37,7 @@ func TestKVPutError(t *testing.T) {
 	defer testutil.AfterTest(t)
 
 	var (
-		maxReqBytes = 1.5 * 1024 * 1024 // hard coded max in v3_server.go
+		maxReqBytes = 1.5 * 1024 * 1024
```

Comment:
Maybe instead of removing, replace it with something like "default max request size"?

Reply:
> Maybe instead of removing, replace it with something like "default max request size"?

Yes, that's what I thought at first. But redefine a constant, not sure which package is better. Therefore, I searched for the occurrence of 1.5m values, which most like my submitted content. If I can determine where to define this constant, I can continue to modify it.

thanks.

Reply:
How about using the definition already existed in embed/config.go?

Reply:
your advise should be the best way at present. It has taken this approach in integration module. I'll try to modify it to see if it's elegant.

Reply:
There are two more substitutions in clientv3/integration that I made at the same time. If it doesn't fit, I'll revise it